rules/scripts/bitstreams_workspace.py

```diff
@@ -258,11 +271,45 @@ def WriteBuildFile(self, build, key):
     name = "bitstream_mask_rom",
     srcs = ["{splice}"],
 )
-""".format(**param))
+""".format(datetime=BitstreamCache._GetDateTimeStr(),
+           key=key,
+           orig=test_rom_file,
+           splice=mask_rom_file)
+
+        used_target_names: Set[str] = set()
+        for mmi_file in files_by_extension.get('mmi', set()):
```

Comment:
Is this overly generic? Or should we genericize it even more and get rid of special casing for the bitstream files?

Reply:
Up to you.  The current structure of the BUILD file is to give convenient labels to files that have what are (imho) inconvenient names.

Reply:
Good point, let's not make this overly generic and lose the convenient naming.